tests/tests_ntp.yml

```diff
@@ -20,20 +20,20 @@
 
   tasks:
     - block:
-      - meta: flush_handlers
+        - meta: flush_handlers
```

Comment:
:+1: for conforming to the style of Ansible docs: https://docs.ansible.com/ansible/latest/user_guide/playbooks_blocks.html